src/get-started/extending-and-modifying-components/index.md.njk

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout-pane.njk
+title: Extending and modifying components in production
+section: Get started
+theme: How to guides
+order: 5
+description: What to do when you want to extend or modify a component in production.
+---
+
+You can extend or modify components in the GOV.UK Design System. You could do this to improve a component based on user research or because you have a specific use case for a component in your service.
+
+If you have a production service consider whether your changes:
+- help the long term maintenance of your service
+- allow you to safely install updates from the GOV.UK Design System
+- reduce the risk of technical debt
+
+When you extend or modify components in the GOV.UK Design System you create potential risk. For example, your code or service may break when you then install Design System updates.
+
+You can help reduce potential risk to your code by:
+
+- using [override classes](#start-with-override-classes)
+- not [overwriting GOV.UK Design System code](#avoid-overwriting-govuk-design-system-code)
+- using a [unique prefix for component names][prefix]
+- creating [custom override classes for multiple components](#custom-override-classes)
+- using [BEM for small modifications to components](#small-modifications-to-components)
+- forking [large modifications to components](#large-modifications-to-components)
+
+We use many of these techniques to make sure our code in the GOV.UK Design System does not unintentionally break your application code.
+
+## Start with override classes
+
+Check the [GOV.UK Design System styles section](/styles/) to see if there’s a CSS class you can use before adding CSS to your service.
+
+For example, you can resolve changes related to:
+
+- spacing with [spacing override classes](/styles/spacing/#spacing-override-classes)
+- font size with [font override classes](/styles/typography/#font-override-classes)
+- width with [width override classes](/styles/spacing/#width-override-classes)
+
+## Avoid overwriting GOV.UK Design System code
+
+If you make a modification involving CSS you might decide to write a selector that targets a GOV.UK Design System class and change its CSS properties.
+
+This will work in the short term but may break if you install an update relying on the component’s previous behaviour.
+
+For example, if you want to override the button component you could do the following
+
+```html
+<div class=”app-interruption-card”>
+  <button class=”govuk-button”>
+    Inverse button
+  </button>
+</div>
+```
+
+```css
+.app-interruption-card .govuk-button {
+  background-color: white;
+  color: blue;
+}
+```
+
+This would work in the short term, but if we changed how the button component works it could then break when you update your service. For this example consider using [small modifications to components](#small-modifications-to-components).
```

Comment:
```suggestion
This would work in the short term, but if the GOV.UK Design System team changes how the button component works it could break when you update your service. For examples like this, consider using [small modifications to components](#small-modifications-to-components).
```